packages/node_modules/@ciscospark/internal-plugin-encryption/test/integration/spec/kms.js

```diff
@@ -91,6 +91,88 @@ describe(`Encryption`, function() {
         }));
     });
 
+    /**
+     *Test listAuthorizations function
+     *Setup: Create a resource, then authorize a user to a key, then authorize a resource to a key
+     *Test:
+     *1)Invoke listAuthorizations on the resource, verify that the function returns the array that contains two authorized items
+     *2)Remove the authorized user from the resource, then invoke listAuthorizations again, verify that the returned array doesn't have the user.
+     *3)Remove the authorized resource from the resource again, then invoke listAuthorizations again, verify that he returned array doesn't have the resource.
+     */
+    describe(`#listAuthorizations()`, () => {
+      let boundedKeyUri, kro, otherKro;
+      const testUserId = mccoy.spark.internal.device.userId;
+      let testResourceId;
+
+      before(`creates a resource`, () => spark.internal.encryption.kms.createUnboundKeys({count: 1})
+        .then(([key]) => spark.internal.encryption.kms.createResource({
+          key
+        }))
+        .then((k) => {
+          kro = k;
+          boundedKeyUri = kro.keyUris[0];
+          assert.lengthOf(kro.authorizationUris, 1);
+        }));
+
+      before(`authorizes a user to a key`, () => spark.internal.encryption.kms.addAuthorization({
+        userIds: [testUserId],
+        kroUri: kro.uri
+      })
+        .then(([auth]) => {
+          assert.equal(auth.resourceUri, kro.uri);
+          assert.equal(auth.authId, testUserId);
+          return assert.isFulfilled(mccoy.spark.internal.encryption.kms.fetchKey({uri: boundedKeyUri}));
+        }));
+
+      before(`authorizes a resource to a key`, () => spark.internal.encryption.kms.createUnboundKeys({count: 1})
+        .then(([key]) => spark.internal.encryption.kms.createResource({key}))
+        .then((k) => {
+          otherKro = k;
+          testResourceId = otherKro.uri;
+          return spark.internal.encryption.kms.addAuthorization({
+            authIds: [testResourceId],
+            kro
+          });
+        })
+        .then(([auth]) => {
+          assert.equal(auth.resourceUri, kro.uri);
+          assert.equal(auth.authId, otherKro.uri);
+          return spark.internal.encryption.kms.listAuthorizations({kroUri: kro.uri})
+            .then((authorizations) => {
+              assert.equal(authorizations.length, 2);
+              assert.include(authorizations, {uri: auth.uri, authId: testUserId, resourceUri: auth.resourceUri});
```

Comment:
assert.include uses strict equality, so this will never hold (since the object defined in parameter 2 isn't in `authorizations`).  If we were on the most recent version of chai, you'd be able to use `assert.deepInclude()`, but since we're not there yet, try something like `assert.include(authorizations.map((a) => a.authId), testUserId`

Reply:
updated accordingly